cobra/flux_analysis/variability.py

```diff
@@ -42,21 +44,35 @@ def _fva_step(reaction_id):
     # in the history manager which can take longer than the actual
     # FVA for small models
     _model.solver.objective.set_linear_coefficients(
-        {rxn.forward_variable: 1, rxn.reverse_variable: -1})
+        {rxn.forward_variable: 1, rxn.reverse_variable: -1}
+    )
     _model.slim_optimize()
     sutil.check_solver_status(_model.solver.status)
     if _loopless:
         value = loopless_fva_iter(_model, rxn)
     else:
         value = _model.solver.objective.value
+    # handle infeasible case
+    if value is None:
```

Comment:
As far as I can tell the value is not set in this condition, so nothing actually changed?

Reply:
Oh yeah, sorry for that. Will fix.